x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/aggregatemetric/AggregateMetricFeatureSetUsage.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.core.aggregatemetric;
+
+import org.elasticsearch.Version;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.xpack.core.XPackFeatureSet;
+import org.elasticsearch.xpack.core.XPackField;
+
+import java.io.IOException;
+import java.util.Objects;
+
+public class AggregateMetricFeatureSetUsage extends XPackFeatureSet.Usage {
+
+    public AggregateMetricFeatureSetUsage(StreamInput input) throws IOException {
+        super(input);
+    }
+
+    public AggregateMetricFeatureSetUsage(boolean available, boolean enabled) {
+        super(XPackField.AGGREGATE_METRIC, available, enabled);
+    }
+
+    @Override public Version getMinimalSupportedVersion() {
+        return Version.V_7_11_0;
```

Comment:
@polyfractal I guess after this PR is merged, we should change the minimal supported version in master branch from `V_8_0_0` to `V_7_11_0`. Right?